package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package>
+  <name>transforms3d</name>
+  <version>0.3.0</version>
```

Comment:
Would it be possible to build this file, with the correct version, as the version gets updated?